packages/gitea/zarf.yaml

```diff
@@ -82,3 +85,5 @@ components:
                 namespace: zarf
                 name: app=gitea
                 condition: Ready
```

Comment:
This wait should probably be first to ensure that the new gitea is up before we start messing with users and tokens